src/arithmetic/algebraic_expression/algebraic_expression.c

```diff
@@ -53,8 +53,9 @@ static AlgebraicExpression *_AlgebraicExpression_CloneOperand
 (
 	const AlgebraicExpression *exp
 ) {
-	return AlgebraicExpression_NewOperand(exp->operand.matrix, exp->operand.diagonal, exp->operand.src,
-										  exp->operand.dest, exp->operand.edge, exp->operand.label);
+	AlgebraicExpression *clone = rm_malloc(sizeof(AlgebraicExpression));
+	memcpy(clone, exp, sizeof(AlgebraicExpression));
+	return clone;
```

Comment:
I made this change because I didn't want to add a new argument to `AlgebraicExpression_NewOperand`, which is already pretty overloaded! I can change it if desired.